src/main/resources/application.properties

```diff
@@ -1,6 +1,3 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/shop?serverTimezone=UTC
```

Comment:
Don't need to remove these properties,  you can use env variable 

Let's see the example 
```
spring.config.import=optional:file:.env[.properties]
spring.datasource.url=jdbc:mysql://localhost:3306/${MYSQLDB_DATABASE}?serverTimezone=UTC
spring.datasource.username=${MYSQLDB_USER}
spring.datasource.password=${MYSQLDB_ROOT_PASSWORD}
spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
spring.jpa.hibernate.ddl-auto=validate
spring.jpa.show-sql=true
server.servlet.context-path=/api
spring.jpa.open-in-view=false

jwt.secret=${JWT_SECRET}
jwt.expiration=${JWT_EXPIRATION}
```